Makefile

```diff
@@ -122,9 +122,10 @@ build: manifests generate fmt vet ## Build manager binary.
 	go build -o bin/manager cmd/main.go
 
 LIGHTSPEED_SERVICE_IMG ?= quay.io/openshift/lightspeed-service-api:latest
+LIGHTSPEED_SERVICE_REDIS_IMG ?= quay.io/vchalla/redis:latest
```

Comment:
we should add a containerfile to the lightspeed-service repo to build this image, and then update our prow job to build + mirror the image, so we don't lose track of this.

Reply:
Acknowledged. Will get that pieces added as well.

Reply:
why not use a more official repo for redis? 
+1 for Ben's advice on dedicated container version file.

Reply:
Modified it to OLS official redis image. Once these PRs are merged it should start working.
* https://github.com/openshift/release/pull/48726
* https://github.com/openshift/lightspeed-service/pull/399